### DISCUSSION TITLE
Team Members

### DISCUSSION THREAD

**Discussion by MamaNira2** [deepgram/community#380] [General help] [answered]
It seems that i am unable to add a new user to a project. While I am able to send them an invitation and they can click on the link, it asks them to log in/create an account. If they create an account it's not connected to the project. Can you please advise how to go about this

**Comment by jpvajda** [comment]
@MamaNira2  Would you be able to provide your Deepgram `project id` so we can look into this issue for you?

We also have our invitations API documented [here. ](https://developers.deepgram.com/reference/list-invites)

If you can clarify if you tried this invite via the API vs the Console UI that be helpful to know.

Finally, how are your users signing up for a new account?

Do they use one of the SSO providers: Google, Github, Azure. OR is it through an email address?

**Comment by MamaNira2** [comment]
@jpvajda Here is the project ID 7a995cb8-b4f3-4e18-b85b-bc724938a2a0
I have tried using the ConsoleUI as i want to be the one that invites rather than having to rely on my team/use API

The users have attempted signing up via email/ through a link using Chrome browser

**Comment by jpvajda** [comment]
@MamaNira2  Thanks for the additional details.

When they try to signup do you happen to know which Identify provider they are using to create their account?

We offer signup via:

- Google
- Github
- Azure
- OR is it through an email address

**Reply by MamaNira2** [reply]
hey @jpvajda  so when I click on the link it opens a new tab in Chrome for me and displays the login page and then we tried with email. its company email and not gmail so we didn't try via Google

![image](https://github.com/deepgram/community/assets/148815883/465b4f27-bd84-4e5e-a9fa-abcf38e08005)

**Comment by jpvajda** [comment]
@MamaNira2  Could you do a test with one of your users? 

1. have them login to the Deepgram Console first.
2. send them an invite to the project.
3. Have them click the invite link via email while they still have an active login session.

Just let me know what transpires after testing this out.

**Reply by MamaNira2** [reply]
@jpvajda we have tested but it still isnt referencing my project, or showing any connection to it. the user can see their own name/project and that is it.

**Reply by jpvajda** [reply]
@MamaNira2  I'm really sorry this isn't working on your end but I'm am unable to reproduce this problem with my own projects. Can you try creating another new project and testing this out once more?  I also spoke to our product engineering team and there are no known issues with invitations on our side.

**Comment by MamaNira2** [comment]
@jpvajda i tried as you suggested, Created new project 39d73ee3-03d2-4669-a29f-9ecbae611e51. Invited someone from IT who has an account with Deepgram. They clicked on the link from the email and this new project isnt showing either. They just have their own project and thats it. While i do believe there might not be know issues on your end, i still need to fix it on my end somehow...

**Comment by jpvajda** [comment]
@MamaNira2 Do you have any networking rules at play here? Are you on a VPN, or does your IT department restrict certain aspects of the Browser?

Looking at the network tab in the Browser Inspector might reveal if something is being blocked in terms of requests to our API. 

One other thought is: 
 
What could be helpful is for you to share with us a video recording of the problem. Could you do a screen recording and send that to our email address so we can continue to evaluate. you can email that to `devrel-at-deepgram.com`

**Comment by MamaNira2** [comment]
@jpvajda I apologize for only responding now i was OOO for a while. Would it be possible for you to provide me with an email address to which I can send you the invite to my project and we can test it that way, you can let me know if it works. Is that ok with you?

**Comment by jpvajda** [comment]
@MamaNira2 Sure. You can send that project invite to `devrel-at-deepgram.com` and I can see what happens when I try to join your project.

**Comment by MamaNira2** [comment]
@jpvajda thank you. But we would have to try a different one. it seems it recognizes @deepgram so it won't allow to send

![image](https://github.com/deepgram/community/assets/148815883/c29355c6-15bd-4066-8acc-df8418f80aba)

**Reply by jpvajda** [reply]
the `at` you used above should be changed to `@` I just didn't type out the entire email here to avoid SPAM Bot scrappers looking for email addresses on the web.

**Reply by MamaNira2** [reply]
@jpvajda i apologize i didn't even realise that... i sent the invite let me know if you can see my project

**Comment by jpvajda** [comment]
@MamaNira2  Thanks! I actually wasn't able to join your project and seem to be experiencing the same issue you are seeing.  Let me do some investigation on our end to see what might be the issue here.

**Reply by MamaNira2** [reply]
@jpvajda thank you so much and sorry for the inconvenience, but this first hand experience seems to be the best explanation i could offer you/come up with. Let me know if you work something out. Thanks

**Comment by jpvajda** [comment]
@MamaNira2 I was hoping you could try to create a brand new project and invite me the same way, this will help us eliminate if there is an unique issue with this particular project, or something else.

**Reply by MamaNira2** [reply]
@jpvajda I just did. Created a test project. However, there is someone named Sandra, successfully in my original project now...

**Reply by jpvajda** [reply]
@MamaNira2  I got into this project without an issue. I think @SandraRodgers  may have tested that initial project invite link and those links expire as soon as they are used which was likely blocking me from joining.


so It looks like this working on our side with invites to your projects.

![Screenshot 2023-12-06 at 3 05 11 PM](https://github.com/deepgram/community/assets/1395158/27055017-04b5-4fb0-9420-4e0ca488adfb)

**Comment by MamaNira2** [comment]
@jpvajda so what do you advise i do? Also do people need to have an account with Deepgram to be able to accept or no? I am a bit confused now

**Comment by jpvajda** [comment, marked as answer]
@MamaNira2  

> do people need to have an account with Deepgram to be able to accept an invite.

No, they should be able to create a new account, THEN accept the invite all at the same time.  However since you are having issues with this workflow, having them setup an account before inviting to the project should make it less steps for them.

We've been unable to reproduce this at this point, so it has to be specific to something on your end, and it's hard for us to know what that is. 

Keep in mind the invite link will expire as soon as the user attempts to accept the invite, so if there is some networking rules or browser rules in your IT domain blocking something it might be the problem.
